__tests__/app.test.js

```diff
@@ -109,3 +108,42 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200 - should return all of the comments with the given article_id", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const response = body.comments;
+        expect(response.length).toBe(11);
+        expect(response).toBeSortedBy("created_at", { descending: true });
+        response.forEach((comment) => {
+          expect(typeof comment.body).toBe("string");
+          expect(typeof comment.article_id).toBe("number");
```

Comment:
I would expect the article_id here to be the exact number since I know it needs to be 1